packages/modular-site/package.json

```diff
@@ -4,10 +4,14 @@
   "license": "Apache-2.0",
   "private": true,
   "dependencies": {
+    "@finos/perspective": "0.6.0",
```

Comment:
I think we can bump these to `0.6.2` to match the current `yarn.lock` versions.

Reply:
I was going for light-touch, so matched it to the version of other `@finos/*` dependencies in `modular-scripts`... do you want them all updating in this package?

Reply:
Done!